docs: updating major upgrade path (#4010)
mesosphere/dcos-docs-site: pages/dkp/kommander/2.1/major-upgrade/upgrade-clusters/migrate-platform-apps/prepare-apps/index.md

#### Alerting

> Custom `alertmanager` [configurations](/dkp/konvoy/1.8/monitoring/#notify-prometheus-alerts-in-slack) must be copied or re-created [manually](/dkp/kommander/2.1/monitoring/#notify-prometheus-alerts-in-slack). You can do this before adapting Prometheus, so that there is no time interval when the `alertmanager` is not configured.

Reviewer: ```suggestion
You must copy or [recreate](/dkp/kommander/2.1/monitoring/#notify-prometheus-alerts-in-slack) custom `alertmanager` [configurations](/dkp/konvoy/1.8/monitoring/#notify-prometheus-alerts-in-slack) before adapting Prometheus, so that there is no time interval when the `alertmanager` is not configured.
```

#### Custom ServiceMonitor objects

> You should copy or re-create ServiceMonitors using the label `release: kube-prometheus-stack` instead of `release: prometheus-kubeaddons`. This can be done before processing, using different names or namespaces for the new objects. Alternatively, you could just re-label any existing ServiceMonitors before or after the processing, but this will introduce an interval where Prometheus is running but the ServiceMonitors have no effect. After the Prometheus-operator in the adapted Prometheus starts, it only looks at ServiceMonitor objects with `release: kube-prometheus-stack` label, and it ignores ServiceMonitors with the old label until they are re-labeled.

Reviewer: ```suggestion
Copy or re-create ServiceMonitors using the label `release: kube-prometheus-stack` instead of `release: prometheus-kubeaddons` before processing, using different names or namespaces for the new objects. Or, you can re-label any existing ServiceMonitors before or after processing, but this will introduce an interval where Prometheus is running but the ServiceMonitors have no effect. After the Prometheus-operator in the adapted Prometheus starts, it only looks at ServiceMonitor objects with `release: kube-prometheus-stack` label, and it ignores ServiceMonitors with the old label until they are re-labeled.
```

> ServiceMonitors created using Helm values of the Addon can be adapted in two ways:

Reviewer 2: ```suggestion
You can adapt the ServiceMonitors created using Helm values of the Addon in two ways:
```

> You can use two ways to adapt ServiceMonitors created using Helm values of the Addon:

Reviewer: Let's put this back please. There are is better than 'you can use'.

Author: @cmastr - the linter stops us from starting a sentence with "there are"

Author: okay - just changed the there are part to the middle, basically: 
ServiceMonitors created using Helm values of the Addon can be adapted in two ways:

Reviewer: dang it!! stupid linter. I like your revision.

Author: :)

> You can use two ways to adapt ServiceMonitors created using Helm values of the Addon:

Reviewer: ```suggestion
There are two ways to adapt ServiceMonitors created using Helm values of the Addon:
```

### Caveats

> -   If you customized the Fluent Bit Addon configuration in 1.x, you must add corresponding configOverrides to your 2.x AppDeployment. The configOverrides should be based on the 2.x defaults combined with your custom modifications.

Reviewer 2: ```suggestion
-   If you customized the Fluentbit Addon configuration in 1.x, you must add corresponding configOverrides to your 2.x AppDeployment. The configOverrides should be based on the 2.x defaults combined with your custom modifications.
```

Author: Fluent Bit goes by Fluent Bit now - two words. 🤷

-   The remaining components of the old logging stack are not uninstalled, and must be manually deleted. See the [post upgrade cleanup](../cleanup) section for details.

-   If your 1.8 cluster only used use Fluent Bit and you do not need other parts of the 2.x logging stack installed, specify the `--skip-new-logging-stack-apps` flag when performing the adaption process.

## Kubernetes-dashboard

Currently, any custom modifications of the addon Helm values are not recognized. The moving process overrides them with Kommander 2.1 values and removes the old addon. Backup your settings and reapply them after processing is complete if required.

## Grafana

You must back up any custom Grafana dashboards before running the adaption process.

## Velero

In 2.x, the Velero install has moved from the `velero` namespace to the `kommander` namespace. If you have runbooks or automation that invokes the velero tool, make sure to update them as suggested below.

> <p class="message--note"><strong>NOTE: </strong> The Velero namespace can be set to <code>kommander</code> with the <code>velero client config set namespace=kommander</code>.  Alternatively you can set the environment variable <code>export VELERO_NAMESPACE=kommander</code>, or use the velero --namespace command line option.</p>

Reviewer 3: ```suggestion
<p class="message--note"><strong>NOTE: </strong> The Velero namespace can be set to <code>kommander</code> with the command <code>velero client config set namespace=kommander</code>. Alternatively you can set the environment variable <code>export VELERO_NAMESPACE=kommander</code>, or use the velero <code>--namespace kommander</code> command line option.</p>
```
